config/initializers/assets.rb

```diff
@@ -12,3 +12,4 @@
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
 # Rails.application.config.assets.precompile += %w( admin.js admin.css )
+Rails.application.config.assets.precompile += %w( devise.css devise.js )
```

Comment:
Can we call it `authentication` instead of `devise`, please?

Reply:
Also, can we get rid of the JS?
```suggestion
Rails.application.config.assets.precompile += %w[authentication.css]
```